modules/admin_manual/pages/installation/manual_installation/manual_installation_db.adoc

```diff
@@ -4,44 +4,34 @@
 
 == Introduction
 
-This document gives you an overview of databases supported by ownCloud. It describes
-some basic installation steps and how to create an administrative user for the database.
-This administrative user is necessary that the ownCloud database and user who further
-manages the ownCloud database can be created The ownCloud database user has no access
+This document gives you an overview of databases supported by ownCloud. It describes some basic installation steps and how to create an administrative user for the database. This administrative user is necessary that the ownCloud database and user who further manages the ownCloud database can be created The ownCloud database user has no access
```

Comment:
```suggestion
This document gives you an overview of databases supported by ownCloud. It describes some basic installation steps and how to create an administrative user for the database. This administrative user is necessary so that the ownCloud database and user who further manages the ownCloud database can be created The ownCloud database user has no access
```